src/vos/tests/vts_ilog.c

```diff
@@ -400,6 +400,49 @@ do_update(daos_handle_t loh, daos_epoch_t epoch,
 	return 0;
 }
 
+struct version_cache {
+	uint32_t	vc_ver[2];
+	int		vc_idx;
+};
+
+static void
+version_cache_init(struct version_cache *vcache, uint32_t initial_version)
+{
+	memset(vcache, 0, sizeof(*vcache));
+	vcache->vc_ver[1] = initial_version;
```

Comment:
Since all the callers use "1" as the initial version, then we can ignore "initial_version" parameter, and set "vcache->vc_ver[1] = 1" directly.

Reply:
done